data_collection/web/advanced.rst

```diff
@@ -24,3 +71,81 @@ The following global names are used by PPAS JavaScript tracker. Websites that wi
 * ``piwik_log``
 * ``piwik_track``
 * ``sevenTag``
+
+
+.. _data-collection-tracker-alternative-configuration:
+
+Alternative multi-domain configurations
+---------------------------------------
+
+Tracking domains and all subdomains
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To track all data between domain and all its subdomains, we must use cookies configured with the following snippet::
+
+    _paq.push(["setTrackerUrl", u+"ppms.php"]);
+    _paq.push(["setSiteId", "XXX-XXX-XXX-XXX-XXX"]);
+
+    // Share the tracking cookie across example.com, www.example.com, subdomain.example.com, ...
+    _paq.push(["setCookieDomain", "*.example.com"]);
+
+    // Tell Piwik the website domain so that clicks on these domains are not tracked as "Outlinks"
+    _paq.push(["setDomains", "*.example.com"]);
+
+    _paq.push(["trackPageView"]);
+
+Tracking multiple domains as one site
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To set up tracking between multiple domains, you must use multiple functions ``setDomains`` to set a list of domains and
+``enableCrossDomainLinking`` to enable cross domain linking::
+
+    _paq.push(["setDomains", domains]);
+
+.. describe:: domains
+
+    **Required** ``array`` Domains array, with wildcards
+
+::
```

Comment:
Description can be removed. We'll link setDomains name to JS tracking client api doc.